requirements.txt

```diff
@@ -27,7 +27,7 @@ salesforce-bulk==2.1.0
 sarge==0.1.5.post0
 selenium==3.141.0
 simple-salesforce==0.74.2  # pyup: <0.74.3
-six==1.12.0
+six==1.13.0
```

Comment:
@davisagli - I don't see any references for `import six` or `from six` in the code base. Looks like this is a py2/3 compatability lib. Are you aware if need this after making the switch to py3?

Reply:
@Br4nd0R we don't use it directly any longer, but it is still a transitive dependency of some of our direct dependencies, so we should continue to include it here to make sure we get a consistent version.

Btw I like using [pipdeptree](https://pypi.org/project/pipdeptree/) to explore the dependency tree.